### DISCUSSION TITLE
Comment soumettre une issue ou feature?

### DISCUSSION THREAD

**Discussion by mathgim** [MyElectricalData/myelectricaldata_import#307] [General]
En voulant soumettre une issue et une feature normalement comme je fais sur d'autres repo, j'ai été bloqué. 
Qu'est ce que j'ai raté ?

![image](https://user-images.githubusercontent.com/60158607/222059300-09016a9a-1602-443e-be4f-60d1ac07d216.png)

**Comment by m4dm4rtig4n** [comment]
C'est un bug dans la CI, mais rien de bloquant.
